pkg/pipeline/extract/aggregate/aggregates.go

```diff
@@ -18,17 +18,26 @@
 package aggregate
 
 import (
-	"fmt"
-	"reflect"
+	"container/list"
+	"sync"
+	"time"
 
 	"github.com/netobserv/flowlogs-pipeline/pkg/api"
 	"github.com/netobserv/flowlogs-pipeline/pkg/config"
+	"github.com/netobserv/flowlogs-pipeline/pkg/pipeline/utils"
 	log "github.com/sirupsen/logrus"
 )
 
+const defaultExpiryTime = 60 * 10 // 10 minutes
+
 type Aggregates []Aggregate
 type Definitions []api.AggregateDefinition
 
+var (
+	mutex      = &sync.Mutex{}
+	expiryTime = int64(defaultExpiryTime)
```

Comment:
Agree - not changing for now just because we have that also in all other places and this will be handled when we generalize the LRU with expiration cache